### PR TITLE
feat: PT-5672 Support parse ipfs schema by URL

### PR DESCRIPTION
**Pull request by sanufriev**

### CODE REVIEW

File: meta-resolver/src/main/java/sun/net/www/protocol/ipfs/Handler.java [hunk 1]
```diff
@@ -0,0 +1,13 @@
+package sun.net.www.protocol.ipfs;
+
+import java.io.IOException;
+import java.net.URL;
+import java.net.URLConnection;
+import java.net.URLStreamHandler;
+
+public class Handler extends URLStreamHandler {
```
Comment (igorbaltiyskiy-rari) [inline meta-resolver/src/main/java/sun/net/www/protocol/ipfs/Handler.java:8]: Why do you need this?

Comment (yehohanan7) [inline meta-resolver/src/main/java/sun/net/www/protocol/ipfs/Handler.java:8]: Minor suggestion: `IPFSHandler`

**Review by yehohanan7 — APPROVED** [review]

File: meta-resolver/src/main/java/sun/net/www/protocol/ipfs/Handler.java (hunk 1 above)
Comment (igorbaltiyskiy-rari) [reply meta-resolver/src/main/java/sun/net/www/protocol/ipfs/Handler.java:8]: It must be named `Handler` for the black magic to work

**Review by igorbaltiyskiy-rari — CHANGES_REQUESTED** [review]
Test fails on Java 17, which is used in prod. Seems like the change is dependent on implementation details that have changed.

**Pull request merged**